code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -490,33 +490,116 @@
-	w_class = SIZE_MEDIUM
-	force = MELEE_FORCE_NORMAL
-	throwforce = MELEE_FORCE_TIER_4
-	inherent_traits = list(TRAIT_TOOL_CROWBAR)
-
-/obj/item/tool/crowbar/maintenance_jack/get_examine_text(mob/user)
-	. = ..()
-	. += SPAN_NOTICE("It is set to [HAS_TRAIT_FROM(src, TRAIT_TOOL_CROWBAR, TRAIT_SOURCE_INHERENT) ? "crowbar" : "wrench"] mode.")
+	w_class = SIZE_LARGE
+	force = MELEE_FORCE_STRONG
+	flags_equip_slot = SLOT_SUIT_STORE
+	pry_capable = IS_PRY_CAPABLE_FORCE //but not really
+	///Whether the Maintenance Jack is on crowbar or wrench mode
+	var/crowbar_mode = TRUE
 
-/obj/item/tool/crowbar/maintenance_jack/attack_self(mob/user)
+/obj/item/maintenance_jack/get_examine_text(mob/user)
 	. = ..()
-	if(HAS_TRAIT_FROM(src, TRAIT_TOOL_CROWBAR, TRAIT_SOURCE_INHERENT))
-		playsound(src, 'sound/weapons/handling/gun_underbarrel_activate.ogg', 25, TRUE)
-		to_chat(user, SPAN_NOTICE("You change your grip on [src]. You will now use it as a wrench."))
+	. += SPAN_NOTICE("Interact with the Maintenance Jack to change modes.")
+	if(crowbar_mode)
+		. += SPAN_NOTICE("It is set to crowbar mode, allowing you to pry open doors, provided you are strong enough. Maybe you could even use this side for surgery...")
+	else
+		. += SPAN_NOTICE("It is set to wrench mode, allowing you to unbolt doors, provided you are smart enough to know how.")
+
+/obj/item/maintenance_jack/attack_self(mob/living/user)
+
+	playsound(src, 'sound/weapons/punchmiss.ogg', 15, TRUE, 3)
+	if(crowbar_mode) //Switch to wrench mode | Remove bolts
+		user.visible_message(SPAN_INFO("[user] changes their grip on [src]. They will now use it as a wrench."),
+		SPAN_NOTICE("You change your grip on [src]. You will now use it as a wrench."))
+		crowbar_mode = FALSE
+		animate(src, transform = matrix(0, MATRIX_ROTATE), time = 2, easing = EASE_IN)
+		animate(transform = matrix(90, MATRIX_ROTATE), time = 1)
+		animate(transform = matrix(180, MATRIX_ROTATE), time = 2, easing = EASE_OUT)
 		REMOVE_TRAIT(src, TRAIT_TOOL_CROWBAR, TRAIT_SOURCE_INHERENT)
 		ADD_TRAIT(src, TRAIT_TOOL_WRENCH, TRAIT_SOURCE_INHERENT)
-	else
-		playsound(src, 'sound/weapons/handling/gun_underbarrel_deactivate.ogg', 25, TRUE)
-		to_chat(user, SPAN_NOTICE("You change your grip on [src]. You will now use it as a crowbar."))
+		pry_capable = null
+
+	else //Switch to crowbar mode | Pry open doors if super strong trait
+		user.visible_message(SPAN_INFO("[user] changes their grip on [src]. They will now use it as a crowbar."),
+		SPAN_NOTICE("You change your grip on [src]. You will now use it as a crowbar."))
+		crowbar_mode = TRUE
+		animate(src, transform = matrix(180, MATRIX_ROTATE), time = 2, easing = EASE_IN)
+		animate(transform = matrix(270, MATRIX_ROTATE), time = 1)
+		animate(transform = matrix(360, MATRIX_ROTATE), time = 2, easing = EASE_OUT)
 		REMOVE_TRAIT(src, TRAIT_TOOL_WRENCH, TRAIT_SOURCE_INHERENT)
 		ADD_TRAIT(src, TRAIT_TOOL_CROWBAR, TRAIT_SOURCE_INHERENT)
+		pry_capable = IS_PRY_CAPABLE_FORCE
+	. = ..()
```

Comment:
can we do . = ..() at the start of the proc and return after the if(crowbar_mode) and removing the else block